### PR TITLE
fix search bar margin/padding on mobile

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: shared/common-adapters/search-filter.tsx [hunk 1]
```diff
@@ -345,8 +345,6 @@ const styles = Styles.styleSheetCreate(() => ({
   containerMobile: Styles.platformStyles({
     common: {
       paddingBottom: Styles.globalMargins.tiny,
-      paddingLeft: Styles.globalMargins.small,
```
Comment (buoyad) [inline shared/common-adapters/search-filter.tsx:348]: prefer to override the style in `profile/search/bar` rather than change all uses of this component

**Comment by joshblum** [comment]
image

File: shared/common-adapters/search-filter.tsx (hunk 1 above)
Comment (buoyad) [reply shared/common-adapters/search-filter.tsx:348]: could we add this style back?

**Comment by chrisnojima** [comment]
- [ ] 🍒

File: shared/common-adapters/search-filter.tsx (hunk 1 above)
Comment (joshblum) [reply shared/common-adapters/search-filter.tsx:348]: oops, forgot to push that

**Pull request merged**